Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -28,21 +28,20 @@ public class DafnyProgramVerifier : IProgramVerifier {
     }
 
     public async Task<IReadOnlyList<IImplementationTask>> GetVerificationTasksAsync(ExecutionEngine boogieEngine,
-      CompilationAfterResolution compilation,
+      Resolution resolution,
```

Comment:
I'm a bit hesitant about the "Resolution" name. Resolution is a process, while class names are usually objects.
What about:
* `ResolutionOutput`
* `ResolutionResult`
* `ResolutionOutcome`
?